internal/atomic.go

```diff
@@ -0,0 +1,19 @@
+package internal
```

Comment:
I think it's better to add `atomic` and `fakepacketconn` (or `fakenet`?) directories